Patches/Nearmare Race/ThingDefs_Misc/Nearmare_Apparel.xml

```diff
@@ -247,6 +247,121 @@
 					<WornBulk>1</WornBulk>
 				</value>
 			</li>
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="HAR_NM_A_EyeCover_a"]/apparel/bodyPartGroups</xpath>
+				<value>
+					<li>Hands</li>
+					<li>Feet</li>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[defName="HAR_NM_Sh_BKo"]/statBases/StuffEffectMultiplierArmor</xpath>

```

Comment:
It's not necessary to remove `StuffEffectMultiplierArmor` tags. Instead, simply change them to the approximate material thickness of the apparel item in mm.

Reply:
StuffEffectMultiplierArmor tag here do not impress thickness of this clothes . So I want to remove it.